src/reducers/registrationReducer.js

```diff
@@ -61,19 +68,23 @@ export default function registration(
     case actions.POST_DISCOUNT_PAYMENT_METHOD_SUCCESS:
       return successSubmit(state);
     case actions.SET_PARENT:
-      newState.CurrentParent = payload;
-      return {...newState};
+      setParentRegistrationCart(payload);
```

Comment:
Reducers are supposed to be [never perform side effects](https://redux.js.org/basics/reducers#handling-actions). Try moving this to the action instead.